### PR TITLE
Add setTopicArn methods to SnsContext and SnsQsContext

### PR DESCRIPTION
**Pull request by gdsmith**
Enables us to prevent #1188

### CODE REVIEW

File: pkg/snsqs/SnsQsContext.php [hunk 1]
```diff
@@ -53,23 +53,15 @@ public function __construct($snsContext, $sqsContext)
         } elseif (is_callable($snsContext)) {
             $this->snsContextFactory = $snsContext;
         } else {
-            throw new \InvalidArgumentException(sprintf(
-                'The $snsContext argument must be either %s or callable that returns %s once called.',
-                SnsContext::class,
-                SnsContext::class
-            ));
+            throw new \InvalidArgumentException(sprintf('The $snsContext argument must be either %s or callable that returns %s once called.', SnsContext::class, SnsContext::class));
```
Comment (andrewmy) [inline pkg/snsqs/SnsQsContext.php:56]: Is there any particular reason for these formatting changes? IMO they reduce readability.

**Comment by andrewmy** [comment]
Pretty sure I saw them before the rebase as well.

**Comment by gdsmith** [comment]
Changes are from `composer cs-fix`. I can revert them if you'd prefer

**Comment by gdsmith** [comment]
Actually, seems like this would fail the build if I revert those changes:

```diff
./bin/php-cs-fixer fix --config=.php_cs.php --no-interaction --dry-run --diff -v --path-mode=intersection -- 'pkg/sns/SnsContext.php' 'pkg/snsqs/SnsQsContext.php'
PHP CS Fixer 2.18.6 Remote Void by Fabien Potencier and Dariusz Ruminski
Runtime: PHP 7.3.11
Loaded config default from ".php_cs.php".
Using cache file "/Volumes/Mukuru/taurus/third-party/enqueue-dev/var/.php_cs.cache".
SF
Legend: ?-unknown, I-invalid file syntax (file ignored), S-skipped (cached or empty file), .-no changes, F-fixed, E-error
   1) /Volumes/Mukuru/taurus/third-party/enqueue-dev/pkg/snsqs/SnsQsContext.php (single_line_throw)
      ---------- begin diff ----------
--- Original
+++ New
@@ @@
         } else {
-            throw new \InvalidArgumentException(sprintf(
-                'The $snsContext argument must be either %s or callable that returns %s once called.',
-                SnsContext::class,
-                SnsContext::class)
-            );
+            throw new \InvalidArgumentException(sprintf('The $snsContext argument must be either %s or callable that returns %s once called.', SnsContext::class, SnsContext::class));
@@ @@
         } else {
-            throw new \InvalidArgumentException(sprintf(
-                'The $sqsContext argument must be either %s or callable that returns %s once called.',
-                SqsContext::class,
-                SqsContext::class)
-            );
+            throw new \InvalidArgumentException(sprintf('The $sqsContext argument must be either %s or callable that returns %s once called.', SqsContext::class, SqsContext::class));
         }
```

**Comment by andrewmy** [comment]
Wow ok :D

**Comment by gdsmith** [comment]
So is this good for deploy and release now 🤞

**Pull request merged**